gateway/models/database.go

```diff
@@ -0,0 +1,33 @@
+package models
+
+import (
+	"database/sql"
+
+	"github.com/hoophq/hoop/common/log"
+	"github.com/hoophq/hoop/gateway/appconfig"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+// This makes the DB generally available to the application
+var DB *gorm.DB
+
+func InitDatabase() {
```

Comment:
We need to make sure this function tries to reconnect in case of a failure,  otherwise the whole application will be unavailable.

Reply:
we don't, if it fails, it fails. This function expects a valid database connection, no magic, just explicitly.